projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts

```diff
@@ -76,11 +77,14 @@ export type OperatorListButtonGroups = Array<{
 export class OperatorListComponent implements OnInit {
 
     static readonly DEFAULT_MIXED_OPERATOR_DIALOGS: Array<OperatorListType> = [
-        // {
-        //     component: RasterValueExtractionOperatorComponent,
-        //     type: RasterValueExtractionType,
-        //     description: 'Attach raster values to vector data',
-        // },
+        {
+            component: RasterVectorJoinComponent,
+            type: {
+                NAME: 'Raster Vector Join',
```

Comment:
we currently only support Raster point join... at least indicate it in the description